content/6_Advanced/DP_More.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 id: dp-more
 title: Additional DP Optimizations and Techniques
-author: Andi Qu
+author: Andi Qu, Thunyatorn Jirachaisri
```

Comment:
don't think this is worth adding your name in authors
feel free to add some more content if you want to keep this